helm/templates/mysql-deployment.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: some-mysql-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: some-mysql
+  template:
+    metadata:
+      labels:
+        app: some-mysql
+    spec:
+      containers:
+        - name: some-mysql
+          image: mysql:latest
+          env:
+            - name: MYSQL_ROOT_PASSWORD
```

Comment:
Secrets haben in einem Git nichts verloren, bitte rausnehmen und entweder in separater .env oder direkt im Secret verwalten

Reply:
Bitte anpassen

Reply:
Request changes

Reply:
Erledigt.